client/src/templates/Challenges/utils/frame.ts

```diff
@@ -124,7 +129,8 @@ const createFrame =
     const frame = document.createElement('iframe');
     frame.id = id;
     if (typeof title === 'string') {
-      frame.title = title;
+      frame.title = i18next.t('misc.iframe-preview');
```

Comment:
```suggestion
      frame.title = i18next.t('misc.iframe-preview', { title });
```

Thank you github for losing my second review comment.